advancedDungeon/src/produsAdvanced/level/AdvancedSortLevel.java

```diff
@@ -184,8 +178,7 @@ protected void onTick() {}
   private void checkPlayerSolution() {
     try {
       solution =
-          ((MonsterSort)
-                  DynamicCompiler.loadUserInstance(MONSTER_SORT_PATH, MONSTER_SORT_CLASSNAME))
+          DynamicCompiler.loadUserInstance(MyMonsterSort.class)
```

Comment:
irgendwie fehlt mir hier noch die angabe "produsAdvanced/riddles", oder? wo soll denn der dynamiccompiler nach "mymonster" suchen gehen? überall im projekt?

Reply:
Das regelt ja der import `import produsAdvanced.riddles.MyMonsterSort;`

Reply:
Okay, dann gehen wir den Step Back.
Bin da nämlich bei dir. Die Review-Wünsche von damals gingen halt in eine andere Richtung. 

Die neue Variante ist halt schön fur konkret unseren Zweck, aber wenn du von extern mit Klassen kommst, ist der spaß vorbei.